php_phongo.c

```diff
@@ -982,11 +982,11 @@ bool phongo_execute_command(mongoc_client_t* client, php_phongo_command_type_t t
 			goto cleanup;
 		}
 
-		cmd_cursor = mongoc_cursor_new_from_command_reply(client, &initial_reply, server_id);
+		cmd_cursor = mongoc_cursor_new_from_command_reply_with_opts(client, &initial_reply, &opts);
 	} else {
 		bson_t* wrapped_reply = create_wrapped_command_envelope(db, &reply);
 
-		cmd_cursor = mongoc_cursor_new_from_command_reply(client, wrapped_reply, server_id);
+		cmd_cursor = mongoc_cursor_new_from_command_reply_with_opts(client, wrapped_reply, &opts);
```

Comment:
Does this change belong in the same commit?

Reply:
It's now split out. Some more work had to be done here — see email to the driver lists.

Reply:
Likely also want to set `serverId` on `opts` here as well.

Reply:
This should also use a `cursor_opts`:

```c
bson_t cursor_opts = BSON_INITIALIZER;
bson_t* wrapped_reply = create_wrapped_command_envelope(db, &reply);


bson_append_int32(&cursor_opts, "serverId", -1, server_id);
cmd_cursor = mongoc_cursor_new_from_command_reply_with_opts(client, wrapped_reply, &opts);
bson_destroy(&cursor_opts);
```

Reply:
updated